docs-site/docs/user/analysis/getting-started.md

```diff
@@ -4,4 +4,49 @@ id: getting-started
+| Automated               | Automated workflow executions belong to projects and once set-up, an analysis is performed on all newly uploaded paired-end files within that project |
+
+User workflow executions are personal and are only accessible to the user that created it, while automated workflow executions are accessible to all project members who have at least the Analyst role.
+
+## Requirements
+
+You will require the following to perform workflow executions:
+  - User workflow execution
+    - Project that has at least one associated sample, and paired-end files uploaded to the sample.
+    - At least the Analyst role for the project containing samples for workflow execution.
+  - Automated workflow execution
+    - A project and at least the Maintainer role for this project.
+
+## Workflow Execution States
+
+While a workflow execution is running, it will go through numerous states to give you feedback on its progress.
+
+  States for a successful workflow execution (in order):
+
+  | State      | Description                                                                                                                 |
+  | :--------- | :-------------------------------------------------------------------------------------------------------------------------- |
+  | New        | This is the initial state acknowledging the workflow execution has successfully created and prepares its files for analysis |
+  | Prepared   | Files were successfully prepared and is being submitted to the selected pipeline                                            |
+  | Submitted  | The workflow execution was submitted to the selected pipeline                                                               |
+  | Running    | The workflow execution is being executed by the selected pipeline                                                           |
+  | Completing | The analysis was successful and IRIDA Next is finalizing the workflow execution  for the user                               |
+  | Completed  | The workflow execution is complete and ready for the user                                                                   |
+
+  Error states:
+
+  | State     | Description                                                                                |
+  | :-------- | :----------------------------------------------------------------------------------------- |
+  | Error     | An error occurred during analysis and the run was aborted                                  |
+  | Canceling | The workflow execution was canceled and IRIDA Next is in the process of canceling the run  |
+  | Canceled  | The workflow execution successfully canceled                                               |
+
+The current state of any workflow execution is shown on the workflow execution [listing page](../analysis/working-with-workflow-executions).
```

Comment:
Could you also add information about deleting a workflow execution.
Specifically
* error/canceled runs being deleted has no additional considerations
* completed runs being deleted does not delete the results that have been propagated to the samples

Reply:
Added in d26ef5bed67f85a18f9ac5c210e6869ba9966f5a. Let me know if the wording wasn't what you expected